data/multiplayer/eras.lua

```diff
@@ -42,25 +42,20 @@ res.turns_over_advantage = function()
 	local income_factor = 5
 
 	local winning_sides = {}
-	local tie = true
 	local total_score = -1
 	local side_comparison = ""
-	local color = "#000000"
+	local winners_color = "#000000"
 	for side, team in all_sides() do
 		if not team.__cfg.hidden then
-			local r, g, b = 255, 255, 255
-			if     team.__cfg.color == 1 then r, g, b = 255,   0,   0
-			elseif team.__cfg.color == 2 then r, g, b =   0,   0, 255
-			elseif team.__cfg.color == 3 then r, g, b =   0, 255,   0
-			elseif team.__cfg.color == 4 then r, g, b = 155,  48, 255
-			elseif team.__cfg.color == 5 then r, g, b =   0,   0,   0
-			elseif team.__cfg.color == 6 then r, g, b = 165,  42,  42
-			elseif team.__cfg.color == 7 then r, g, b = 255, 165,   0
-			elseif team.__cfg.color == 8 then r, g, b = 255, 255, 255
-			elseif team.__cfg.color == 9 then r, g, b =   0, 128, 128 end
+			-- The translatable strings include support for coloring the side names, but here we just use white. The team colors accessible to Lua
+			-- could be "red", "blue", etc (which are also supported by Pango's markup), but could also be user-defined names from a color changer mod.
+			-- Logged as bug #5722.
+			local side_color = "#ffffff"
```

Comment:
You know, the API needed to just directly get the colour already exists…

```lua
local side_color = string.format("#%02x%02x%02x", table.unpack(wesnoth.colors[team.color].mid))
```

Or if you uncomment https://github.com/wesnoth/wesnoth/tree/master/src/scripting/lua_color.cpp#L121-L125:

```lua
local side_color = wesnoth.colors[team.color].pango_hex
```

Reply:
Thanks. Using `pango_hex` looks like a nice encapsulation, but I think it should be done in a separate PR.